app/views/home.html

```diff
@@ -1 +1,15 @@
-<h1>Hi this is home page</h1>
+<h1>Pokédex Tracker</h1>
+
+<div class="hero">
+  <img class="mockup" src="mockup.png" width="700px">
+  <p>A tool for tracking your <a href="http://bulbapedia.bulbagarden.net/wiki/Living_Pok%C3%A9dex" target="_blank" class="link">Living Dex</a> progress. This project's source code is open source, available on Github (<a href="https://github.com/robinjoseph08/api.pokedextracker.com" target="_blank" class="link">API</a> &amp; <a href="https://github.com/robinjoseph08/pokedextracker.com/" target="_blank" class="link">website</a>). Feel free to contribute or leave comments or issues!</p>
```

Comment:
do you think it might be worthwhile to also link to a user's dex so that people can see what a progressing one looks like before they commit and sign up?